### ISSUE TITLE
`from x import y as z` causes crash

### ISSUE THREAD

**Issue by charmoniumQ** [lyz-code/autoimport#162]
## Description
See title

## Steps to reproduce
1. `echo from subprocess import run as run > test.py`
2. `autoimport test.py`

## Current behavior
```
Traceback (most recent call last):
  File "/home/sam/.cache/pypoetry/virtualenvs/charmonium.determ-hash-sYmCxUo--py3.9/bin/autoimport", line 8, in <module>
    sys.exit(cli())
  File "/home/sam/.cache/pypoetry/virtualenvs/charmonium.determ-hash-sYmCxUo--py3.9/lib/python3.9/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/home/sam/.cache/pypoetry/virtualenvs/charmonium.determ-hash-sYmCxUo--py3.9/lib/python3.9/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/home/sam/.cache/pypoetry/virtualenvs/charmonium.determ-hash-sYmCxUo--py3.9/lib/python3.9/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/sam/.cache/pypoetry/virtualenvs/charmonium.determ-hash-sYmCxUo--py3.9/lib/python3.9/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home/sam/.cache/pypoetry/virtualenvs/charmonium.determ-hash-sYmCxUo--py3.9/lib/python3.9/site-packages/autoimport/entrypoints/cli.py", line 19, in cli
    fixed_code = services.fix_files(files)
  File "/home/sam/.cache/pypoetry/virtualenvs/charmonium.determ-hash-sYmCxUo--py3.9/lib/python3.9/site-packages/autoimport/services.py", line 27, in fix_files
    fixed_source = fix_code(source)
  File "/home/sam/.cache/pypoetry/virtualenvs/charmonium.determ-hash-sYmCxUo--py3.9/lib/python3.9/site-packages/autoimport/services.py", line 67, in fix_code
    return SourceCode(original_source_code).fix()
  File "/home/sam/.cache/pypoetry/virtualenvs/charmonium.determ-hash-sYmCxUo--py3.9/lib/python3.9/site-packages/autoimport/model.py", line 59, in fix
    self._fix_flake_import_errors()
  File "/home/sam/.cache/pypoetry/virtualenvs/charmonium.determ-hash-sYmCxUo--py3.9/lib/python3.9/site-packages/autoimport/model.py", line 254, in _fix_flake_import_errors
    self._remove_unused_imports(import_name)
  File "/home/sam/.cache/pypoetry/virtualenvs/charmonium.determ-hash-sYmCxUo--py3.9/lib/python3.9/site-packages/autoimport/model.py", line 405, in _remove_unused_imports
    imports.remove(object_name)
ValueError: list.remove(x): x not in list
```
I did some basic debugging. The pattern on L397 does not expect `as z` part of the statement. On L404, `imports = ['x as z']`. L405 attempts to remove `z` from this list.

## Desired behavior
No crash

## Environment
autoimport 0.7.5

**Comment by lyz-code** [comment]
Thank you for taking the time for opening the issue, and good catch! Seeing that you've already looked into the issue, could you make a PR to fix it? I'd appreciate it a lot :)

If you can't whenever I have some time left, I'll change it myself

**Comment by charmoniumQ** [comment]
Yeah, I'd be happy to, but it may be a while for me.

**Comment by lyz-code** [comment]
Hey @charmoniumQ this should be fixed in `0.9.1`. If it doesn't work I'll reopen the issue
